test/unit_test/test_general_functions.py

```diff
@@ -0,0 +1,78 @@
+from lexos.helpers.general_functions import get_encoding, make_preview_from, \
+    generate_d3_object, merge_list, load_stastic, matrix_to_dict, \
+    dict_to_matrix, html_escape, apply_function_exclude_tags, decode_bytes
+import unittest
+
+
+class TestGeneralFunctions(unittest.TestCase):
```

Comment:
we actually do not use `unittest` framework of python, therefore inheriting `TestCase` will be unnecessary. Do you have any reason we should switch to `unittest` framework?